.circleci/config.yml

```diff
@@ -163,7 +163,7 @@ jobs:
   # and to make more clear when the job fails
   check-golangci-lint:
     docker:
-      - image: golangci/golangci-lint:v1.33.2
+      - image: golangci/golangci-lint:v1.38.0
```

Comment:
why not use 1.39.0? which is the latest

Reply:
No reason, it's been a while since the PR was opened. We can update ;)

Reply:
i will do in a followup, lets get this merged